### ISSUE TITLE
OtlpHTTPExporter HTTP/JSON support? 

### ISSUE THREAD

**Issue by jhoongo** [open-telemetry/opentelemetry-swift#465]
Not sure if I'm missing something here, but it looks like `OtlpHttpExporter` is serializing the data as [the protocol buffer binary format](https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/Exporters/OpenTelemetryProtocolHttp/OtlpHttpExporterBase.swift#L33). 

Is there any plan to support HTTP/JSON requests for the OTLP HTTP Exporter?

**Comment by hunguyenaxon** [comment]
I'm working around by using HTTP/Protobuf requests, but it only works if your OTel server accept that format.

**Comment by bryce-b** [comment]
Yes, the http exporter is intended to be using protobuf. There is currently an active PR working on an otlp json exporter (#480)
